patchwork/api/patch.py

```diff
@@ -280,9 +309,7 @@ def update(self, instance, validated_data):
             if instance.related and instance.related.patches.count() == 2:
                 instance.related.delete()
             instance.related = None
-            return super(PatchDetailSerializer, self).update(
-                instance, validated_data
-            )
+            return
```

Comment:
Is this correct? Could you add a comment explaining why we're returning without calling `super()`?